.gitignore

```diff
@@ -215,3 +215,4 @@ fabric.properties
 # .idea/misc.xml
 # *.ipr
 .idea/
```

Comment:
1. You can delete all other .idea/<file-name> entries in line 168-181
2. Please also include `.DS_STORE` and `.vscode`

Reply:
Thanks! Updated